src/lib/drizzle-helpers.js

```diff
@@ -0,0 +1,48 @@
+import { sql } from 'drizzle-orm'
+import { assert } from '../utils.js'
+/** @import { BetterSQLite3Database } from 'drizzle-orm/better-sqlite3' */
+
+/**
+ * @param {unknown} queryResult
+ * @returns {number}
+ */
+const getNumberResult = (queryResult) => {
+  assert(
+    queryResult &&
+      typeof queryResult === 'object' &&
+      'result' in queryResult &&
+      typeof queryResult.result === 'number',
+    'expected query to return proper result'
+  )
+  return queryResult.result
+}
+
+/**
+ * Get the number of rows in a table using `SELECT COUNT(*)`.
+ * Returns 0 if the table doesn't exist.
+ *
+ * @param {BetterSQLite3Database} db
+ * @param {string} tableName
+ * @returns {number}
+ */
+export const tableCountIfExists = (db, tableName) =>
+  db.transaction((tx) => {
+    const existsQuery = sql`
+      SELECT EXISTS (
+        SELECT 1
+        FROM sqlite_master
```

Comment:
Maybe use `sqlite_schema`? https://sqlite.org/schematab.html

Reply:
When you do `PRAGMA table_list` you see `sqlite_schema` as the only table.

Reply:
`sqlite_master` is an "historical" alias for `sqlite_schema`. I should be using `_schema`. Fixed in 574dca554b057d4f822c4279f5c47c6e0fa208d2.